src/index.js

```diff
@@ -205,21 +205,33 @@ export default class {
 
     const collectionsToSync = collections || Object.keys(this.collections)
     let collectionsActiveSync = {}
+    let docs = {}
+    let percentage = 0
+    let syncedData = 0
 
     const handleOnSync = (syncState, collectionName, moduleByName) => {
+      const runOnSync = (percentage, syncedData, collectionsActiveSync) => {
+        this.sync.onSync && this.sync.onSync(percentage, syncedData, collectionsActiveSync)
+        moduleByName.sync && moduleByName.sync.onSync && moduleByName.sync.onSync(percentage, syncedData, collectionsActiveSync)
+      }
+
+      syncState.change$.subscribe(change => {
+        docs[collectionName] = change.change ? change.change.docs_read : change.docs_read
+        syncedData = Object.values(docs).reduce((acc, act) => acc + act, 0)
```

Comment:
Vamos tentar não abreviar os nomes
```suggestion
        syncedData = Object.values(docs).reduce((accumulator, currentValue) => accumulator + currentValue, 0)
```